libraries/libfc/test/CMakeLists.txt

```diff
@@ -1,12 +1,13 @@
-add_subdirectory( crypto )
-add_subdirectory( io )
-add_subdirectory( network )
-add_subdirectory( scoped_exit )
-add_subdirectory( static_variant )
-add_subdirectory( variant )
-add_subdirectory( variant_estimated_size )
+file(GLOB UNIT_TESTS
```

Comment:
Not a huge fan of `file(GLOB)` since it's easy to be confused why a new .cpp file isn't being compiled. But now that we're on cmake 3.12+ we can use `CONFIGURE_DEPENDS` which makes it less of a trap.

Also, I wonder about using `GLOB_RECURSE` here so we don't need to specify each sub directory?

Reply:
I think it would have been interesting to try `CONFIGURE_DEPENDS` + `GLOB_RECURSE` and see how it goes. Up to you.